src/ColorVectorSpace.jl

```diff
@@ -314,6 +314,10 @@ end
 (.*){T<:Number}(A::AbstractArray{T}, b::TransparentGray) = mul(A, b)
 (.*){T<:Number}(b::TransparentGray, A::AbstractArray{T}) = mul(b, A)
 
+varm{C<:AbstractGray}(v::AbstractArray{C}, s::AbstractGray; corrected::Bool=true) =
+        varm(map(gray,v),gray(s); corrected=corrected)
+real{C<:AbstractGray}(x::Type{C}) = C
```

Comment:
i'm just trying to get `var` to work on an `Image`, and it needs `real` defined on the type.  from base/statistics.jl:

```
function var{T}(A::AbstractArray{T}; corrected::Bool=true, mean=nothing)
    convert(real(momenttype(T)),
            mean == 0 ? varzm(A; corrected=corrected) :
            mean === nothing ? varm(A, Base.mean(A); corrected=corrected) :
            isa(mean, Number) ? varm(A, mean::Number; corrected=corrected) :
            throw(ArgumentError("invalid value of mean, $(mean)::$(typeof(mean))")))::real(momenttype(T))
end
```

happy to make the changes you suggest, but `Gray{Float32} <: Real == false` for me even on the master branch of ColorVectorSpace (ie, w/o my new definition of `real`)